bitnami/kafka/values.yaml

```diff
@@ -1698,13 +1701,44 @@ provisioning:
   ##
   waitForKafka: true
 
+## @section Kraft chart parameters
+
+## Kraft configuration
+## Kafka mode without Zookeeper. Kafka nodes can work as controllers in this mode.
+##
+kraft:
+  ## @param kraft.enabled Switch to enable or disable the Kraft mode for Kafka
+  ##
+  enabled: false
+  ## @param kraft.controllerPort Kafka Controller listener port
+  ##
+  controllerPort: 9095
+  ## @param kraft.processRoles Roles of your Kafka nodes. Nodes can have 'broker', 'controller' roles or both of them.
+  ##
+  processRoles: broker,controller
+  ## @param kraft.controllerListenerNames Controller listener names
+  ##
+  controllerListenerNames: CONTROLLER
+  ## @param kraft.clusterId Kafka ClusterID. You must set it if your cluster contains more than one node.
+  ## Generate with `cat /proc/sys/kernel/random/uuid | tr -d '-' | base64 | cut -b 1-22`. Run `export LC_ALL=C` before if you generate it on MacOS.
+  ## Example: k2yipv1sRue7z2_Y3o976A
+  ##
+  clusterId: ""
+  ## @param kraft.controllerQuorumVoters Quorum voters of Kafka Kraft cluster. Use it for nodes with 'broker' role only.
+  ## Example: 1@controller1.example.com:9095,2@controller2.example.com:9095
+  ##
+  controllerQuorumVoters: ""
+
+
+
 ## @section ZooKeeper chart parameters
 
 ## ZooKeeper chart configuration
 ## https://github.com/bitnami/charts/blob/main/bitnami/zookeeper/values.yaml
 ##
 zookeeper:
   ## @param zookeeper.enabled Switch to enable or disable the ZooKeeper helm chart
+  ## Must be false if you use Kraft mode.
```

Comment:
To include that information in the README file please move this sentence to the end of the upper line.